src/api/local/deployTools.ts

```diff
@@ -57,7 +57,7 @@ export namespace DeployTools {
         if (!method) {
           const methods = [];
           if (Deployment.getConnection().remoteFeatures.md5sum) {
-            methods.push({ method: "compare" as DeploymentMethod, label: `Compare`, description: `Synchronizes using MD5 hash comparison` });
+            //methods.push({ method: "compare" as DeploymentMethod, label: `Compare`, description: `Synchronizes using MD5 hash comparison` });
```

Comment:
I guess this wasn't meant to be commited?
```suggestion
            methods.push({ method: "compare" as DeploymentMethod, label: `Compare`, description: `Synchronizes using MD5 hash comparison` });
```